src/caselawclient/models/documents/__init__.py

```diff
@@ -70,7 +63,7 @@ class Document:
 
     attributes_to_validate: list[tuple[str, bool, str]] = [
         (
-            "failed_to_parse",
+            "is_failure",
```

Comment:
Reasoning about this change is hard: this feels like potentially a major breaking change and struggling to review it.

Reply:
`failed_to_parse` is now implemented in `DocumentBody`, not `Body`. `is_failure` currently proxies this and only this, but was left as an abstraction because it could potentially be expanded in future to include other things we consider to be a total failure state.

By making the `Document` validity checks care only about this abstraction method it's less likely that future changes to what we consider a failure would be missed.